patches/fraction.js+4.3.7.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/node_modules/fraction.js/package.json b/node_modules/fraction.js/package.json
+index 085d287..94061ea 100644
+--- a/node_modules/fraction.js/package.json
++++ b/node_modules/fraction.js/package.json
+@@ -19,14 +19,15 @@
+     "email": "robert@raw.org",
+     "url": "https://raw.org/"
+   },
+-  "type": "module",
++  "type": "commonjs",
+   "main": "fraction.cjs",
+   "exports": {
+     ".": {
+       "import": "./fraction.js",
+       "require": "./fraction.cjs",
+       "types": "./fraction.d.ts"
+-    }
++    },
++    "./bigfraction.js": "./bigfraction.js"
```

Comment:
So: I changed nothing about fraction.js that seemed material, but CI complained of being unable to import when I pushed to GH. This was frustrating, but it did seem that the complaint on the CI side was justified. Maybe there is a difference between the shadow-cljs version on my machine vs. what CI is running, but I thought that it would be safer to fix this problem "once and for all" (see below)

Reply:
obsolete